File: TestWebApp/TestingApp/Models/BoughtArticle.cs
using static TestingApp.Extensions;

namespace TestingApp.Models
{
    public class BoughtArticle
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int PriceId { get; set; }
        public Price Price { get; set; }
        public int Amount { get; set; }
        public byte[] ImageData { get; set; }
        public ArtType Type { get; set; }
    }
}

File: TestWebApp/Bezahlwebsite/ClientApp/src/Interfaces/BoughtArticle.ts
import { ArtType } from "./Enums";
import { Price } from "./Price";

export interface BoughtArticle {
  id: number;
  name: string;
  priceId: number;
  price: Price;
  amount: number;
  imageData: string;
  type: ArtType;
  active: boolean;
  isInVending: boolean;
  vendingSlot: number;
  vendingMachineNumber: number;
}
